runtime/v1/linux/runtime.go

```diff
@@ -392,6 +394,16 @@ func (r *Runtime) loadTasks(ctx context.Context, ns string) ([]*Task, error) {
 			go io.Copy(ioutil.Discard, shimStderrLog)
 		}
 
+		go func() {
```

Comment:
Would it be better to close these after the `io.Copy` operations unblock?

Reply:
+1

Reply:
or put the go goroutine at the beginning to make sure that the log can be closed.

Reply:
@crosbymichael  the io.Copy still block after shim exit, so close it directly after shim exit. or wait it with timeout?

Reply:
it should unblock